packages/react-core/src/components/Pagination/OptionsToggle.tsx

```diff
@@ -59,41 +59,37 @@ export const OptionsToggle: React.FunctionComponent<OptionsToggleProps> = ({
   toggleTemplate: ToggleTemplate,
   onEnter = null
 }: OptionsToggleProps) => (
-  <div
-    className={css(
-      styles.optionsMenuToggle,
-      isDisabled && styles.modifiers.disabled,
-      styles.modifiers.plain,
-      styles.modifiers.text
-    )}
-  >
+  <>
     {showToggle && (
-      <React.Fragment>
-        <span className={css(styles.optionsMenuToggleText)}>
```

Comment:
Products could be applying custom styles here .  This would be a breaking change.